packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```
@@ -0,0 +1,34 @@
+{
+  "pci_database_add_title": "Créer un serveur de données",
+  "pci_database_add_description1": "Créer une base de données disponible dans votre projet Public Cloud. Si vous souhaitez utiliser un réseau privé et que celui-ci n’existe pas encore, merci de",
+  "pci_database_add_description2": "créer un réseau privé",
+  "pci_database_add_description3": "ou un",
+  "pci_database_add_description4": "sous-réseau",
+  "pci_database_add_description5": "d’abord.",
+  "pci_database_add_step1_title": "Choisissez votre base de données",
+  "pci_database_add_step1_description": "Sélectionner la base de données correspondant à vos besoins",
+  "pci_database_add_step2_title": "Sélectionner une offre",
+  "pci_database_add_step3_title": "Sélectionner une localisation",
+  "pci_database_add_step4_title": "Caractéristiques",
+  "pci_database_add_nodes_count": "Nombre de noeuds",
+  "pci_database_add_nodes_count_tooltip": "Le nombre de noeuds varie en fonction de l’offre de service choisie. Pendant la beta, ce nombre ne peut pas être modifié",
+  "pci_database_add_select_flavor": "Sélectionner la saveur des nœuds",
```

```suggestion
  "pci_database_add_select_flavor": "Sélectionner le modèle des nœuds",
```
Saveur means "flavor" as something you can taste 😆

---

Thanks. Its done